src/transaction/log_applier.c

```diff
@@ -7262,7 +7295,7 @@ la_log_page_check (const char *database_name, const char *log_path, INT64 page_n
       free_and_init (la_Info.log_data);
     }				/* check_copied_info */
 
-check_copied_info_end:
+check_copied_log_page_info_end:
```

Comment:
I think it should return error instead of NO_ERROR

Reply:
It should be set NO_ERROR because -p option will not affect printing `printf ("\n *** Delay in Copying Active Log *** \n");`
3387L in the util_cs.c

Reply:
So, why do you check the error at 3313L ?

Reply:
there are two purposes.

1. set the check_copied_info and it will be used `printf ("\n *** Delay in Copying Active Log *** \n");`
2. -r option is checked after 3313L. So, get the error message, error is checked.

Reply:
I think the code is too hard to understand.
I don't know why the method of operation is so different even though it is inside the same function. 
In some labels, if an error occurs, an error is returned, 
and in this case, the error is ignored even if an error occurs.

And even the label name is ambiguous. I think `copied_info` can include `copied_log_page_info`. 
I think `copied_info` should be separated into two stages. 
The step of preprocessing, and the step of getting log page information. 
(e.g. `copied_log_volume/header_info`, `copied_log_page_info`)

what do you think about it? 

If you want to return NO_ERROR for any reason,
please leave a comment on why it is returning NO_ERROR despite the error.

Reply:
1. 

> I think the code is too hard to understand.

> If you want to return NO_ERROR for any reason,
please leave a comment on why it is returning NO_ERROR despite the error.

Its purpose is to handle outside of this function.(for the applyinfo())
-L option affects outside of this function (printf ("\n *** Delay in Copying Active Log *** \n");) but -p option does not.

2. I think `copied_info` should be separated into two stages.
I understand your suggestion is -L option should be separated into two stages. Could you explain more?

Reply:
1. Yes, I know your purpose. But will the intent be easy to communicate to anyone reviewing, analyzing, and maintaining this code? please, make the code readable or leave a comment at this line why you returning the NO_ERROR. 
2. I said about the label. `copied_info` and `copied_log_page_info` are used as labels now. but it seems ambiguous. because `copied_info` includes the `copied_log_page_info`.